README.md

```diff
@@ -19,4 +19,5 @@ For more information, please see the links below.
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python?ref=badge_large)
 
-test
+test2
```

Comment:
The change from 'test' to 'test2' lacks context and purpose. Consider providing a more meaningful description or removing this line if it's not serving any specific purpose in the README.